src/components/MainLayout/MainPage/CommonGameArea/GameBoard/GameBoardPart/PanelRow/WeatherPart.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+import { colors } from '../../../../../../../theme/variables';
+import gameAreaBlock from '../../../../../../../assets/battlefield/field-item.png';
+
+export const WeatherPart = styled.div`
+  width: 100%;
+  height: 95%;
+  margin: 0.1em 0 0.1em 0;
+  justify-content: left;
+  align-items: left;
+  border-left: 0.2em solid grey;
+  background-image: url(${gameAreaBlock});
+  background-repeat: no-repeat;
+  background-size: cover;
+  &:hover {
```

Comment:
we don't need to highlight this area by hover, the player only interacts with the cards

the illumination of the area occurs by the system itself, i.e. it is recommended to throw a prop of the form "highlited=true/false" into the component to show this part only when prop is true

Reply:
Do I need to delete the highlight?

Reply:
it would be better to add prop and set it to the false state, when we have the allocation logic, we will use it

Reply:
Ok